### ISSUE TITLE
[BUG] Error in loading configuration. Error:open /config/resource_config.yaml: no such file or directory

### ISSUE THREAD

**Issue by edgarpf** [infracloudio/botkube#229]
**Describe the bug**

Message error in Kubernetes: **Error in loading configuration. Error:open /config/resource_config.yaml: no such file or directory**

**To Reproduce**
1. I downloaded the develop branch.
2.  I built the image using **make container-image** and I pushed to the dockerhub.
3. Run botkube in kubernetes with my custom image that is exacly a copy of the develop repository.
3. Error in kubernetes: **Error in loading configuration. Error:open /config/resource_config.yaml: no such file or directory**.

Go version was 10.04

**Comment by PrasadG193** [comment]
Hi @edgapf,
Could you please tell me how are you installing BotKube from develop branch? Which command are you using?

**Comment by edgarpf** [comment]
As I said I simple downloaded develop branch and I used **make container-image** to generate an image. I pushed it to dockerhub (**edgarpf/botkube**) and I tried to use the image in kubernetes but I found the error.

**Comment by PrasadG193** [comment]
As a part of https://github.com/infracloudio/botkube/pull/226, we have changed the configuration format little bit. But unfortunately, due to another issue, we have to revert that commit. I'll update the README accordingly once we merge that PR again

**Comment by edgarpf** [comment]
Ok. Thanks.

**Comment by edgarpf** [comment]
I did the same process again and I got the same result with branch develop. But It worked with branch master.

**Comment by PrasadG193** [comment]
Have you pulled latest changes from develop?

**Comment by edgarpf** [comment]
Yes.

**Comment by PrasadG193** [comment]
Did you try removing the older image before you build newer one?

**Comment by PrasadG193** [comment]
Also, can you please tell me how are you deploying it on K8s cluster?

**Comment by PrasadG193** [comment]
If you are using helm, please set version as `9.99.9-dev` if you are deploying from develop branch.

**Comment by PrasadG193** [comment]
Also, make sure that you have pushed the newer image to dockerhub :slightly_smiling_face:

**Comment by edgarpf** [comment]
As I said it worked like a charm using the master branch. I should have used that branch since the beginning because it is the stable version. My fault. I did again all the steps described above after the rollback in the develop branch.

I did not remove the old image but I tagged the new image and pushed this new image to dockerhub. 

I was using **deploy-all-in-one.yaml** with my custom image to deploy botkube. 

Anyway, the problem is solved for me.  Issue closed.
